configs/next/packages/expat/sources

```diff
@@ -21,7 +21,7 @@
 
 ATSRC_PACKAGE_NAME="Expat XML Parser"
 ATSRC_PACKAGE_VER="2.2.5"
-ATSRC_PACKAGE_REV=50f4e1695fda
+ATSRC_PACKAGE_REV=39e487da353b
```

Comment:
The version number needs an update, as shown by:

$ git show 39e487da353b:expat/CMakeLists.txt | grep PACKAGE_VERSION
set(PACKAGE_VERSION "2.2.6")

or

$ git describe 
R_2_2_6